### DISCUSSION TITLE
VipsDisp 2.5.1 compiles but does not run with VIPS 8.15.0 rc1 release

### DISCUSSION THREAD

**Discussion by vvinycity** [libvips/libvips#3728] [Q&A]
I compiled VipsDisp 2.5.1 latest sources with new version of libvips (vips-dev-8.15.0 rc1).

The executable does not run and gives an error:
"Procedure entry point g_regex_new could not be located in the dynamic link library libglib-2.0-0.dll"

So, it seems that libglib-2.0-0.dll in 8.15.0 rc1 release is outdated.

Where could I find libglib-2.0-0.dll with "g_regex_new" function defined?

Thank you for your help.

**Comment by jcupitt** [comment]
Is this on Windows? 

I've never tried build vipsdisp on windows, there are probably some linking bugs. Please open an issue on vipsdisp and we can dig into it.

**Reply by vvinycity** [reply]
Thank you for your answer.
Yes, this is on Windows. Still, it is unlikely that this is a linking bug: g_regex_new entry is truly absent in a version of liglib-2.0-0.dll included in 8.15 rc1 binary package. Please, see below. I opened the issue regarding this problem.

<img width="1085" alt="g_regex_new_absent" src="https://github.com/libvips/libvips/assets/8726996/7c1347db-7728-4897-a1ba-9e27245c7237">

**Reply by jcupitt** [reply]
See https://github.com/jcupitt/vipsdisp/issues/23
